apps/src/templates/sectionsRefresh/SectionsSetUp.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {useState} from 'react';
+import i18n from '@cdo/locale';
+import SetUpSectionForm from './SetUpSectionForm';
+import Button from '@cdo/apps/templates/Button';
+import moduleStyles from './sections-refresh.module.scss';
+
+const useSections = () => {
+  const [sections, setSections] = useState([{}]);
+
+  const updateSection = (sectionIdx, keyToUpdate, val) => {
+    const newSections = sections.map((section, idx) => {
+      if (idx === sectionIdx) {
+        return {
+          ...section,
+          [keyToUpdate]: val
+        };
+      } else {
+        return section;
+      }
+    });
+    setSections(newSections);
+  };
+
+  return [sections, updateSection];
+};
+
+export default function SectionsSetUp() {
+  const [sections, updateSection] = useSections();
+  return (
+    <div>
+      <h1>{i18n.setUpClassSectionsHeader()}</h1>
+      <p>{i18n.setUpClassSectionsSubheader()}</p>
+      <p>
+        <a href="code.org">{i18n.setUpClassSectionsSubheaderLink()}</a>
```

Comment:
is this intentionally a placeholder link?

Reply:
Yes -- I need to check with Tess what the link should be. We might already have the page

Reply:
^ @tess323

Reply:
I think https://www.youtube.com/watch?v=4Wugxc80fNU is the best we have right now.  As part of this we should re-write or rerecord a help article/video given the process will change